src/main/java/com/logsentinel/util/StringUtil.java

```diff
@@ -52,4 +52,23 @@ public static String join(String[] array, String separator) {
         }
         return out.toString();
     }
+
+    /**
+     * Add the missing padding of a Base-64 string.
+     *
+     * @param base64 Base64-encoded string
+     * @return padded Base-64 string
+     */
+    public static String base64StringAddPadding(String base64) {
```

Comment:
why do we need this? The server-returned base64 should be okay. Or it isn't?

Reply:
The server returns Base-64 encoded data without the needed padding.